src/components/com_tjdashboard/administrator/languages/en-GB/en-GB.com_tjdashboard.ini

```diff
@@ -149,4 +149,5 @@ COM_TJDASHBOARD_WIDGET_FORM_DESC_PARAMS = "You can add widget params here"
 ;Widgets List view
 COM_TJDASHBOARD_DASHBOARDS_FILTER_SEARCH_DESC = "Search in widget title.Prefix with ID:to search for a widget ID"
 
-
+;dashboards view
+COM_TJDASHBOARD_DASHBOARDS_DELETE_ERROR_MESSAGE = 'Something Went wrong OR first delete Widgets under this dashboard'
```

Comment:
Change this to

Could not delete the dashboard. If this dashboard has any widgets, please delete the widgets before deleting the dashboard

Reply:
Yes. The message that I proposed reflects that.

Reply:
done